aws-android-sdk-s3/build.gradle

```diff
@@ -7,10 +7,22 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 29
+        // Required when setting minSdkVersion to 20 or lower
+        multiDexEnabled true
+    }
+
+    compileOptions {
+        // Flag to enable support for the new language APIs
+        coreLibraryDesugaringEnabled true
+        // Sets Java compatibility to Java 8
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Ahh, right. We have this stuff enabled in Amplify, but I guess it isn't in the `aws-sdk-android`. This seems like a lot to add for one small feature. We might want to just skip the try-with-resources altogether, and just do it the old-fashioned way:
```java
OutputStream outStream = new FileOutputStream(file);
try {
    useTheResource(outStream);
} catch (IOException ioException) {
    // Dang, it didn't work out.
} finally {
    outStream.close();
}
```

Reply:
Got it! 😄